app/src/main/res/layout/entry_submission_row.xml

```diff
@@ -42,7 +42,7 @@
             android:layout_width="@dimen/entry_submission_image_size"
             android:layout_height="match_parent"
             android:layout_alignParentRight="true"
-            android:background="@drawable/camera_icon"
+            android:background="@android:color/black"
```

Comment:
both were temporary placeholders. I just changed it to black for the demo. We should probably have a designated placeholder image. I'm not sure if the contest image endpoint is up and ready yet.